doc/modules/multiclass.rst

```diff
@@ -440,47 +334,66 @@ Below is an example of multiclass learning using Output-Codes::
       Hastie T., Tibshirani R., Friedman J., page 606 (second-edition)
       2008.
 
-Multioutput regression
-======================
-
-Multioutput regression support can be added to any regressor with
-:class:`MultiOutputRegressor`.  This strategy consists of fitting one
-regressor per target. Since each target is represented by exactly one
-regressor it is possible to gain knowledge about the target by
-inspecting its corresponding regressor. As
-:class:`MultiOutputRegressor` fits one regressor per target it can not
-take advantage of correlations between targets.
-
-Below is an example of multioutput regression:
-
-  >>> from sklearn.datasets import make_regression
-  >>> from sklearn.multioutput import MultiOutputRegressor
-  >>> from sklearn.ensemble import GradientBoostingRegressor
-  >>> X, y = make_regression(n_samples=10, n_targets=3, random_state=1)
-  >>> MultiOutputRegressor(GradientBoostingRegressor(random_state=0)).fit(X, y).predict(X)
-  array([[-154.75474165, -147.03498585,  -50.03812219],
-         [   7.12165031,    5.12914884,  -81.46081961],
-         [-187.8948621 , -100.44373091,   13.88978285],
-         [-141.62745778,   95.02891072, -191.48204257],
-         [  97.03260883,  165.34867495,  139.52003279],
-         [ 123.92529176,   21.25719016,   -7.84253   ],
-         [-122.25193977,  -85.16443186, -107.12274212],
-         [ -30.170388  ,  -94.80956739,   12.16979946],
-         [ 140.72667194,  176.50941682,  -17.50447799],
-         [ 149.37967282,  -81.15699552,   -5.72850319]])
-
-Multioutput classification
-==========================
-
-Multioutput classification support can be added to any classifier with
-:class:`MultiOutputClassifier`. This strategy consists of fitting one
-classifier per target.  This allows multiple target variable
+Multilabel classification
+=========================
+
+**Multilabel classification** (closely related to **multioutput**
+**classification**) is a classification task labelling each sample with ``x``
```

Comment:
```suggestion
**classification**) is a classification task labeling each sample with ``x``
```

Reply:
There were a couple of instances of this -- I think I've caught them all. Thanks much! :)